app/components/add-datafruit.js

```diff
@@ -19,6 +19,10 @@ export default class AddDatafruit extends Component {
   @tracked
   datafruit = null;
 
+  get currentTimestamp() {
+    let timestamp = new Date(this.currentDatafruit.createdAt);
```

Comment:
the issue seems to be that `currentDatafruit` might be `undefined` here

Reply:
thanks! tests are passing now. I was wondering if it should return anything if there isn't a microtext yet 🤔

Reply:
oops that causes a lint error anyways. posted a fix